server/src/utils/fight/fightMethods.ts

```diff
@@ -379,7 +385,7 @@ const healFighter = (
   updateStats(stats, fighter.id, 'hpHealed', amount);
 };
 
-const increaseInitiative = (fighter: DetailedFighter) => {
+const increaseInitiative = (fighter: DetailedFighter, ease: number = 1) => {
```

Comment:
ease is specific for checks, it doesn't make sense for initiative.
```suggestion
const increaseInitiative = (fighter: DetailedFighter, multiplicator: number = 1) => {
```